js/bybit.js

```diff
@@ -6904,7 +7660,7 @@ module.exports = class bybit extends Exchange {
          * @method
          * @name bybit#fetchTransfers
          * @description fetch a history of internal transfers made on an account
-         * @see https://bybit-exchange.github.io/docs/account_asset/v3/#t-querytransferlist
+         * @see https://bybit-exchange.github.io/docs/v5/asset/inter-transfer-list
```

Comment:
@sc0Vu something is off here:
![image](https://user-images.githubusercontent.com/43336371/220913138-45ce333a-e40c-4676-a867-21768225623b.png)